pkg/client/interface.go

```diff
@@ -39,7 +39,7 @@ type MsgCreateClaim interface {
 	cosmostypes.Msg
 	GetRootHash() []byte
 	GetSessionHeader() *sessiontypes.SessionHeader
-	GetSupplierAddress() string
```

Comment:
If you do a non-case sensitive search for `supplieraddr` or `supplier addr` a few others come up.

I think it's a good reference to do a sanity check IF we need to add (for clarity) owner/operator there.

![Screenshot 2024-08-07 at 1 02 15 PM](https://github.com/user-attachments/assets/8af05d16-86f3-4d37-bc05-761ac4eeff58)
![Screenshot 2024-08-07 at 1 02 10 PM](https://github.com/user-attachments/assets/4ddf5962-8ed2-47f2-9aaa-a30973956977)